### ISSUE TITLE
Create a 1.8 branch

### ISSUE THREAD

**Issue by JackDn** [bergerhealer/TrainCarts#70]
I'm about to submit a pull request because of #41 but you already started updating TrainCarts to 1.9 in the master branch (and in a 1.9 branch too). Because of this there is no way to release new features for 1.8.

TL;DR It would be nice if you added a 1.8 branch or reverted b4b317e1bcc1da6ea753784c97332936ff7db9f4 to allow adding features to the 1.8 version.

**Comment by mg-1999** [comment]
Master branch is still sort of 1.8. I am trying to revert it back to 1.8 so
we can develop without breaking the 1.8 branch.

Op zo 15 mei 2016 08:37 schreef Jack <notifications@github.com>:

> I'm about to submit a pull request because of #41
> <https://github.com/bergerhealer/TrainCarts/issues/41> but you already
> started updating TrainCarts to 1.9 in the master branch (and in a 1.9
> branch too). Because of this there is no way to release new features for
> 1.8.
>
> TL;DR It would be nice if you added a 1.8 branch to be able to add
> features to the 1.8 version.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bergerhealer/TrainCarts/issues/70>
>
-- 

Met vriendelijk groeten,

Matthijs Giezenberg

**Comment by mg-1999** [comment]
Yeah I was working on 1.9 and cleaned the master branch up. Looking forward to the PR

**Comment by JackDn** [comment]
I already created a PR (#71). I tested it locally after reverting your commit and everything worked fine.
